test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTests.cs

```diff
@@ -33,7 +33,9 @@ public class MvcTagHelpersTests
         // Testing SelectTagHelper with Html.BeginForm 
         [InlineData("CreateWarehouse", null)] 
         // Testing the HTML helpers with FormTagHelper
-        [InlineData("EditWarehouse", null)] 
+        [InlineData("EditWarehouse", null)]
```

Comment:
what changed on this line?  if you removed trailing whitespace, fine.  otherwise revert changes to this line